demo/sovereign/const-rollup-config/src/lib.rs

```diff
@@ -1,4 +1,4 @@
 /// The namespace used by the rollup to store its data. This is a raw slice of 8 bytes.
 /// The rollup stores its data in the namespace b"sov-test" on Celestia. Which in this case is encoded using the
 /// ascii representation of each character.
-pub const ROLLUP_NAMESPACE_RAW: [u8; 4] = [3, 0, 0, 0];
+pub const ROLLUP_NAMESPACE_RAW: [u8; 16] = 3_u128.to_be_bytes();
```

Comment:
FYI comment is really wrong

Reply:
Yep, the entire demo is littered with those comments. Should be addressed in one fell swoop.